src/components/ContextualMenu/ContextualMenuDropdown/ContextualMenuDropdown.test.tsx

```diff
@@ -17,15 +17,39 @@ describe("ContextualMenuDropdown ", () => {
   describe("adjustForWindow", () => {
     const generateFits = (overrides = {}) => {
       const fits = {
-        fromTop: { fitsAbove: true, fitsBelow: true },
-        fromBottom: { fitsAbove: true, fitsBelow: true },
-        fromLeft: { fitsLeft: true, fitsRight: true },
-        fromRight: { fitsLeft: true, fitsRight: true },
+        fromTop: {
```

Comment:
Some of the tests mock the return values for the hook so they needed to be updated to include the new spacing values.